packages/airnode-admin/src/implementation.ts

```diff
@@ -206,13 +256,15 @@ export async function setWhitelistExpiration(
   airnodeAddress: string,
   endpointId: string,
   requesterAddress: string,
-  expirationTimestamp: number
+  expirationTimestamp: number,
+  overrides?: ethers.Overrides
 ) {
   const tx = await requesterAuthorizerWithAirnode.setWhitelistExpiration(
     airnodeAddress,
     endpointId,
     requesterAddress,
-    expirationTimestamp
+    expirationTimestamp,
+    { ...overrides }
```

Comment:
I think we can just pass `overrides` (we don't need to destructure)

Reply:
I used destructuring here because the SDK wrapped methods would pass in `overrides`  as `undefined` if they were omitted, e.g.
```js
  sponsorRequester = (requesterAddress: string, overrides?: ethers.Overrides) =>
    admin.sponsorRequester(this.airnodeRrp, requesterAddress, overrides); // omitted overrides are undefined when passed in to admin.sponsorRequester
```
Ethers seems to throw an error if overrides are undefined, but an empty object works.

Please correct me if I'm wrong or you would have a better solution here.